src/config.py

```
@@ -341,19 +442,77 @@ class CacheSettings(HonchoSettings):
+    # DeductionSpecialist: handles logical inference + temporal reasoning
+    DEDUCTION_MODEL: str = "anthropic/claude-haiku-4.5"
+    # InductionSpecialist: identifies patterns across observations
+    INDUCTION_MODEL: str = "anthropic/claude-haiku-4.5"
+
+    @model_validator(mode="after")
+    def _validate_token_budgets(self) -> "DreamSettings":
+        """Ensure the output token limit exceeds the thinking budget."""
+        if self.MAX_OUTPUT_TOKENS <= self.THINKING_BUDGET_TOKENS:
+            raise ValueError(
+                "MAX_OUTPUT_TOKENS must be greater than THINKING_BUDGET_TOKENS"
+            )
```

Ruff (0.14.10)

Avoid specifying long messages outside the exception class

(TRY003)